WorkloadManagementSystem/DB/tests/Test_JobDB.py

```diff
@@ -0,0 +1,36 @@
+""" tests for the JobDB module """
+
+#pylint: disable=protected-access, missing-docstring
+
+import unittest
+from mock import MagicMock, patch
+
+from DIRAC import S_OK
+
+MODULE_NAME = "DIRAC.WorkloadManagementSystem.DB.JobDB"
+
+class JobDBTest( unittest.TestCase ):
+
+  def setUp( self ):
+
+    def mockInit(self ):
+      self.log = MagicMock()
+      self.logger = MagicMock()
```

Comment:
why this?

Reply:
https://github.com/DIRACGrid/DIRAC/blob/f529123e8453735b22ef4067e0ce5c91d96ab308/Core/Utilities/MySQL.py#L388